memprof/memtrace/Makefile

```diff
@@ -73,17 +73,22 @@ ocamlopt :=  $(BEST_OCAMLOPT) $(flags)
 
 ocamldep := $(BEST_OCAMLDEP)
 depflags := -slash
-depincludes :=
+depincludes := -I $(ROOTDIR)/driver
 
 .PHONY: all allopt opt.opt # all, allopt and opt.opt are synonyms
 all: ocamlopt-memtrace.opt
 allopt: all
 opt.opt: allopt
 
-ocamlopt-memtrace.opt: $(native_modules)
+ocamlopt-memtrace.opt: $(ROOTDIR)/compilerlibs/ocamlcommon.cmxa \
+                       $(ROOTDIR)/compilerlibs/ocamloptcomp.cmxa \
+                       $(ROOTDIR)/driver/optmaindriver.cmx \
```

Comment:
This line should be removed, `.depend` correctly tracks this dependency with your patch

Reply:
done